package.json

```diff
@@ -20,7 +20,9 @@
     "test:integration-legacy": "cucumber-js --fail-fast",
     "test:integration": "jest --config jest.config.integ.js --passWithNoTests",
     "test:protocols": "yarn build:protocols && lerna run test --scope '@aws-sdk/aws-*'",
-    "local-publish": "node ./scripts/verdaccio-publish/index.js"
+    "local-publish": "node ./scripts/verdaccio-publish/index.js",
+    "lint": "eslint 'packages/**/src/*.ts' --fix",
+    "prettify": "prettier --write './packages/**/src/*.ts'"
```

Comment:
```suggestion
```

> We don't have prettify function implemented

Although there is not separate prettify script, we prettify on save and on precommit. So, a separate prettify script is not required.
* Code to run prettier on Save https://github.com/aws/aws-sdk-js-v3/blob/e6a4fcff20560d9302032e6e9c6f3ebae91478b2/.vscode/settings.json#L3-L4
* Code to run prettier on precommit https://github.com/aws/aws-sdk-js-v3/blob/e6a4fcff20560d9302032e6e9c6f3ebae91478b2/package.json#L91-L99

Reply:
Okay removed script

Reply:
Running prettier on precommit will be another MR